backoffice/src/Repositories/ImageRepository.cs

```diff
@@ -70,6 +69,12 @@ public async Task ImagesToDeleteForWreckMaterialAsync(Guid wmId, IEnumerable<Gui
         var images = await Context.Images
             .Where(image => image.WreckMaterialId == wmId && !imagesToKeep.Contains(image.Id))
             .ToListAsync();
+        
+        //Remove images from s3 bucket
+        foreach ( var image in images )
```

Comment:
I'd do this after removing from db, just in case there's an error deleting from the db we want to keep the link intact 

Also rename images to something like imagesToDelete